bitcoin/src/sign_message.rs

```diff
@@ -126,6 +125,15 @@ mod message_signing {
             })
         }
 
+        /// Create from a byte slice.
+        #[deprecated(since = "0.32.0", note = "Use `from_byte_array` instead.")]
+        pub fn from_slice(bytes: &[u8]) -> Result<MessageSignature, MessageSignatureError> {
+            // Convert the Vec<u8> into a fixed-length array
```

Comment:
No need for this comment. For your understanding, this is whats referred to as a "what" comment, it comments what the code does, and hence adds no value because we can read the code. The comments we do use are "why" comments, why the code is doing what its doing (in instances when the why is not obvious).

By "adds no value" I mean adds no value to someone who fully knows the language, and we tend to optimize for those readers, the reason being if we pick some other group of devs to optimize for how would we know which group, week 1 devs, year 2 devs, year 10 devs?

Reply:
Removed and noted for future, thanks.